### ISSUE TITLE
Zwave Error Messages: Unhandled value type

### ISSUE THREAD

**Issue by Olivier6767** [domoticz/domoticz#3348]
Domoticz 4.10929 (latest) and Open-Zwave latest build (1.6).

When Zwave starts I get the following error messages:
```
2019-06-24 15:31:33.555 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.556 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.556 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.557 Error: OpenZWave: Unhandled value type: 4, OpenZWave.cpp:2151
2019-06-24 15:31:33.557 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.560 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.560 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.560 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:31:33.560 Error: OpenZWave: Unhandled value type: 1, OpenZWave.cpp:1689
2019-06-24 15:33:00.261 Error: OpenZWave: Unhandled value type: 4, OpenZWave.cpp:2151
2019-06-24 15:33:00.261 Error: OpenZWave: Value_Changed: Tried adding value, not succeeded!. Node: 56 (0x38), CommandClass: DOOR LOCK, Label: Locked (Advanced), Instance: 1, Index: 1
```

Yesterday I added a Qubino ZMNHAD1 and I got more of the above error messages.
This new module does not work well, for example temperature readings only come in when Zwave is started, they do not update regularly like they should (module settings are correct).

Also when looking at the ZWave Groups, this new module is not linked to Group1 (controller) like all the others. When trying to add it it will not work (timeout message). 

Also, when opening the OZW Control Panel, it is empty, does not show any nodes any more. I will open another bug for this.

**Comment by petergebruers** [comment]
Is this a production network or a test network?

**Comment by Olivier6767** [comment]
This is a production network.

**Comment by petergebruers** [comment]
Then I recommend going back to the stable version. There is still some more work to do to make Domoticz fully compatible with OpenZWave 1.6.

**Comment by Olivier6767** [comment]
I tried going back to the stable version (with open-zwave 1.4) but I get link errors because I have boost libraries 1.70 (which is requirement for the DEV version). I guess I have to go back to a previou sversion of boost?
Can I have 2 version of boost libraries at the same time?

**Comment by Fishwaldo** [comment]
I just tested a ZMNHAD1 H1S6P2 and it worked fine (although I don't have a Temp Sensor Probe). You need to ensure that the controller is part of Group 9 according to the Manual.

**Comment by Olivier6767** [comment]
I confirm that the switch function works fine. However the Temperature report only comes in when I restart ZWave or Domoticz completely. It is not updated anymore after that.

**Comment by petergebruers** [comment]
Even with your controller added to group 9? As @Fishwaldo pointed out, the description of that group says "Group 9: multilevel sensor report (triggered at change of temperature sensor) up to 16 nodes." - so it is possible you don't get (unsolicited) updates if controller is only in group 1.

**Comment by Olivier6767** [comment]
I don't know how to add the controller to Group 9. 
I also compared with another Qubino device that works well. The only difference is that the other one has 2 switches instead of one. It is a ZMNHBDx model. I also have a temperature sensor on this one and it updates correctly when temperature changes. The documentation for this one mentions Group 8 for MultiSensor report (temperature). However I did not do anything specific to add Group 8 to the controller, in fact I have no idea how to do this. 
All mu devices are linked to Group 1 be default and this works. However when looking at the ZWave Network Groups, the newly added Qubino (ZLNHADx) is not linked to Group 1. When I try to add it Group 1, nothing happens...
I believe this is why it does not report the temperature correctly.
Node 61 is the new one (ZMNHADx) and node 59 is ZMNHBDx.

![image](https://user-images.githubusercontent.com/28684355/60259440-8c416f80-98d7-11e9-9f91-258f27067235.png)

Any idea on how to fix this?

**Comment by petergebruers** [comment]
> I don't know how to add the controller to Group 9.

If you click on "+" in column "Groupe 9" you can enter a number, try:

1.1

Then click on "refresh" at the bottom.

If that does not work, repeat but enter:

1

And refresh



> All mu devices are linked to Group 1 be default and this works.

I actually expect more recent devices to have "1.1" and older ones to have "1" - but indeed something should be there in *some* group. Your screenshot shows no groups fo node 61 and that is odd indeed! 

On non-Z-Wave-Plus devices the most important group is not necessarily "Group 1". On Z-Wave Plus the call Group 1 "lifeline". Still no guarantee though that *all* reports go to group 1. OpenZWave detects Z-Wave plus devices and auto adds lifeline.

I cannot test this, I don't own the device.

Question. If you use the physical switch on the device itself, does Domoticz notice the change (after you refresh that page)? Because if it sees the change, that means a "1" is in at least one of those groups and then it is a display issue in Domoticz. If you don't get switch status, nor temperature status, then it is indeed possible that your controller is not in any group.

Did you try the "Refresh Node Info" button in Domoticz?

**Comment by Olivier6767** [comment]
When adding 1.1 in Group9 or Group 1, the command seems to be accepted (no error) and I see this in the log file : 
`2019-06-27 13:19:54.887 Status: OpenZWave: added node: 1 (0x01) instance 1 in group: 9 of node: 61 (0x3d)`

However even after refresh the Groups stay empty in the Domoticz display.

I cannot easily test the use of the physical switch as it is not easily accessible. However when I use domoticz to trigger the switch (on and off) this works without delay.

I did try the 'Refresh Node Info' button but no change.

**Comment by Olivier6767** [comment]
Ok , I was able to make some progress.
I created a new server with Domoticz master branch (stable), along with open-zwave 1.4.
I connected the ZWave controller to this machine and started Domoticz with the default database (I could not start with a copy of the current DB because the DB version is higher).
It created new devices for all my ZWave hardware. 
At this point the node 61 (ZMNHADx) had the connections to Node 1  in groups 1, 2 and 9. Temperature readings were updating regularly.
Also the OZW Control Panel was working fine.

![image](https://user-images.githubusercontent.com/28684355/60275306-02ef6480-98fa-11e9-9371-472d1b4183f8.png)

I then connected the ZWave controller back to my current (up to date) Domoticz server.
The Group associations for Node 61 were kept in the new version (Groups 1, 2 and 9 mapped to Node 1) and the temperature now updates correctly.

However the OZW Control Panel still does not work (empty content) in the current version...
